### PR TITLE
add 3 UCLA faculty

### PR DESCRIPTION
**Pull request by harrygxu**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by emeryberger** [comment]
This diff is too large; probably a line ending issue. Please fix and I will merge.

File: csrankings.csv [hunk 1]
```diff
@@ -8987,6 +8986,7 @@ Miguel P. Correia,Universidade de Lisboa,http://www.gsd.inesc-id.pt/~mpc,X4ejY-k
 Miguel Pupo Correia,Universidade de Lisboa,http://www.gsd.inesc-id.pt/~mpc,X4ejY-kAAAAJ
 Miguel Valero-García,Polytechnic University of Catalonia,http://epsc.upc.edu/projectes/usuaris/miguel.valero,NOSCHOLARPAGE
 Miguel Ángel Zamora Izquierdo,University of Murcia,http://webs.um.es/mzamora/miwiki/doku.php,EQqywLsAAAAJ
+Mihaela van der Schaar,University of California - Los Angeles,DZ3S--MAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:8989]: missing web page URL

File: csrankings.csv [hunk 2]
```diff
@@ -10787,6 +10787,7 @@ Ravi Chugh,University of Chicago,http://people.cs.uchicago.edu/~rchugh,DKuq2fIAA
 Ravi Janardan,University of Minnesota,http://umn.edu/home/janardan,NOSCHOLARPAGE
 Ravi K. Iyer,Univ. of Illinois at Urbana-Champaign,https://www.ece.illinois.edu/directory/profile/rkiyer,NOSCHOLARPAGE
 Ravi Mukkamala,Old Dominion University,http://www.cs.odu.edu/~mukka,NOSCHOLARPAGE
+Ravi Netravali,University of California - Los Angeles,4C9naMgAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:10790]: as above, missing home page URL

File: csrankings.csv [hunk 3]
```diff
@@ -12360,7 +12361,8 @@ Stefano Ceri,Politecnico di Milano,http://home.deib.polimi.it/ceri,aiulJmIAAAAJ
 Stefano Chessa,University of Pisa,http://www.di.unipi.it/~ste,KHO57CAAAAAJ
 Stefano Ermon,Stanford University,https://cs.stanford.edu/~ermon,NOSCHOLARPAGE
 Stefano Lonardi,University of California - Riverside,http://www.cs.ucr.edu/~stelo,T1tFq2MAAAAJ
-Stefano Ricciardi,University of Molise,http://docenti.unimol.it/index.php?u=s.ricciardi,
+Stefano Ricciardi,University of Molise,http://docenti.unimol.it/index.php?u=s.ricciardi
+Stefano Soatto,University of California - Los Angeles,lH1PdF8AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:12365]: and here too

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by harrygxu** [comment]
Files updated

File: csrankings.csv (hunk 3 above)
Comment (harrygxu) [reply csrankings.csv:12365]: Added

File: csrankings.csv (hunk 2 above)
Comment (harrygxu) [reply csrankings.csv:10790]: Added

File: csrankings.csv (hunk 1 above)
Comment (harrygxu) [reply csrankings.csv:8989]: Added

**Review by emeryberger — APPROVED** [review]

**Pull request merged**